tidb-configuration-file.md

```diff
@@ -352,6 +352,11 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + TiDB 在 TCP 层开启 keepalive。
 + 默认值：true
 
+### `tcp-no-delay`
+
++ TiDB 在 TCP 层开启 no delay。
```

Comment:
+ Could you introduce the benefits of enabling no delay in the TCP layer?
+ Is this config for the version later than v5.0 GA?

Reply:
> Could you introduce the benefits of enabling no delay in the TCP layer?

I think https://github.com/pingcap/tidb/pull/22757#issuecomment-782790927 has answered this problem that mysql provides this option, and TiDB introduces it will be compatible with mysql. and According to mysql doc and Best Practices for TCP Optimization in 2019, i think using Nagle's algorithm is useful for user in some specific situation.

Do you want me to add the benefits to the doc?

> Is this config for the version later than v5.0 GA?

Sorry, I do not know the plan, maybe https://github.com/pingcap/docs/pull/5070#issuecomment-804592553 answered this.

Reply:
> Could you introduce the benefits of enabling no delay in the TCP layer?

IMHO we can describe that higher performance can be expected by enabling it, and put the link for [Nagle's algorithm](https://en.wikipedia.org/wiki/Nagle%27s_algorithm).

> Is this config for the version later than v5.0 GA?

Yes, I believe so.

@TomShawn @HuGanghui

Reply:
@HuGanghui Any update?